operator/integration-tests/model-in-image/mii-sample-wrapper/generate-sample-doc.sh

```diff
@@ -152,10 +152,11 @@ for phase in initial update1 update2 update3 update4; do
   # setup ingress yaml files
   $SCRIPTDIR/stage-and-create-ingresses.sh -dry
 
-  for type in WLS JRF; do
+  for IMAGE_TYPE in WLS WLS-CM JRF; do
```

Comment:
To complete support for JRF in the CM case: add "JRF-CM" to the for loop, add a corresponding template, rerun this script (expect mismatch failure), check-in the newly generated sample files into the expected sample location, rerun (expect a pass this time).  The main test script should then pass for the '-jrf -cm' case.